pkg/endpoint/bpf.go

```diff
@@ -63,19 +64,21 @@ func (e *Endpoint) writeL4Map(fw *bufio.Writer, owner Owner, m policy.L4PolicyMa
 		}
 
 		redirect = common.Swab16(redirect)
-		entry := fmt.Sprintf("{%d,%d,%d}", dport, redirect, protoNum)
+		entry := fmt.Sprintf("%d,%d,%d,%d", index, dport, redirect, protoNum)
 		if array != "" {
 			array = array + "," + entry
 		} else {
 			array = entry
 		}
 
+		index += 1
```

Comment:
I think @aanm and I both missed that this is a map and not slice. The current code is OK